block.go

```diff
@@ -96,6 +97,21 @@ func (bc *BlockClient) Get(ctx context.Context, id BlockID) (Block, error) {
 	return decodeBlock(response)
 }
 
+// Delete https://developers.notion.com/reference/retrieve-a-block
```

Comment:
Seems to point to the block retrieval reference instead of to the delete one:
https://developers.notion.com/reference/delete-a-block

Reply:
Is it possible to modify it on your end, or do I need to send another PR ?